pandas/core/internals/managers.py

```diff
@@ -1604,14 +1609,23 @@ def _blklocs(self):
         """ compat with BlockManager """
         return None
 
+    def getitem_mgr(self, indexer) -> SingleBlockManager:
```

Comment:
shouldn't this exist for BlockManager and just  be no-op?

Reply:
its only ever used by Series